src/datastore.ts

```diff
@@ -288,6 +288,14 @@ export class Datastore extends Component implements IDataContext {
       .ofKind("move_category")
       .projected((entry) => entry.value);
   }
+
+  get moveRulesets(): StandardIndex<DataswornTypes["move_ruleset"]> {
+    this.assertReady();
+    return this.indexer.prioritized
+      .ofKind("move_ruleset")
+      .projected((entry) => entry.value);
```

Comment:
A more direct way of producing a map of id to ruleset would be to use the rulesetid source tag:

https://github.com/iron-vault-plugin/iron-vault/blob/main/src/datastore/data-indexer.ts#L11-L21